### ISSUE TITLE
Frequency tables for multiple statistics

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#3391]
Why not be able to have frequency tables able to show multiple statistics, e.g. counts and percentages.  Summary tables allows this.

**Comment by dannyparsons** [comment]
Would need a bit of modification of our methods but possible

**Comment by lilyclements** [comment]
@rdstern I can see we currently give the option to give counts or percentages -- but not both (and only the percentage by one factor. Not an option to go beyond that!).

1. How about we modify this dialog - and code - and have the option to give
* Counts
* Percentages
* Counts, with Percentages in brackets? -- e.g., like "`7 (15%)`"

2. At the moment, if percentages is checked, then you can give it as an overall percentage, or the percentage of a variable. But what if you want to give multiple percentages?
Could we instead have two checkboxes: One saying "Overall percentages", and the other saying "Factor percentages"
If overall is checked, then you get the overall percentage
If factor percentage is checked, then there is a **multiple** receiver, and you can get multiple percentages -- e.g., you want two percentages for each variety-fertgrp combination: The variety percentage, and the fertgrp percentage. 

![Image](https://github.com/user-attachments/assets/2f5c6778-d46c-4dc9-9576-2d1c5e0ea05f)

**Comment by rdstern** [comment]
@lilyclements I really like your suggestions and would like to put them into a more general context.  Then maybe continue our sort of current attack or strategise in parallel.

Just to start by adding a further option to your suggestions above.  When giving tables of percentages I quite like the option of being able to replace the margin of 100%, by giving the counts in that margin!

Now also back to your request for more options in this dialog.  I think this is the right place, but note that we also have the Describe > 2/3 Variables > Summarise where we I think just give one percent, and could opt for more - maybe there in a set of simpler tables.  And we do give multiple percentages (rather poorly) in the Describe > 2/3 Variables > Frequencies  dialog.

In a broader context percentages (and alternatives) is an important topic that is often misunderstood and used badly.  For example, for PICSA we give fractions out of 10 instead.

I suggest this is maybe the third fruit uncovered in your trawl of items - partly related to teaching - where we could have some quick wins, and could work on fairly quickly.

a) Finish analyses for ordered factors - really just our improved Likert graphs to go.
b) Include interaction graphs
c) Better percentages for teaching and use.

And we have quite a big development team to keep occupied!  

Reflecting on these, so far, your suggestions here look maybe like the "lowest hanging fruit".  And would fit well with maybe Anastasia wanting to become more involved in using R-Instat?

**Comment by lilyclements** [comment]
@rdstern I agree with this - and perhaps we should discuss a plan-of-action on this next week.
To add - earlier today I was in a meeting where we wanted our percentages to be a factor of two columns - e.g., we wanted the percentage by the variety _and_ village combination, rather than two percentages with one by variety and one by village.

**Comment by rdstern** [comment]
@lilyclements agreed.  Let's first complete the current round of presentation tables.  I hope this can be done soon now.  Then - maybe even as soon as July (this year!) can we return to this area.  There are still some possible improvements in the 2/3 variable summarise and in the 2/3 variable graph.  I would also include these multiple precentages here.
